src/main/java/io/cdap/plugin/google/drive/source/GoogleDriveInputFormat.java

```diff
@@ -18,6 +18,9 @@
 
 import com.github.rholder.retry.RetryException;
 import com.google.api.services.drive.model.File;
+import com.google.gson.JsonObject;
```

Comment:
Why are these imports added?

Reply:
These imports were left behind after we did some additional tests. Thanks for letting me know. I removed them.